### ISSUE TITLE
App crashes when trying to report a crash

### ISSUE THREAD

**Issue by pacien** [pacien/tincapp#67]
Irony.

```
E/AndroidRuntime: FATAL EXCEPTION: main
                  Process: org.pacien.tincapp, PID: 763
                  java.lang.RuntimeException: Failure from system
                      at android.app.Instrumentation.execStartActivity(Instrumentation.java:1675)
                      at android.app.ContextImpl.startActivity(ContextImpl.java:917)
                      at android.app.ContextImpl.startActivity(ContextImpl.java:888)
                      at android.content.ContextWrapper.startActivity(ContextWrapper.java:379)
                      at org.pacien.tincapp.context.App$Companion.sendMail(App.kt:82)
                      at org.pacien.tincapp.context.App$Companion.sendMail$default(App.kt:74)
                      at org.pacien.tincapp.activities.common.RecentCrashHandler.sendReportMail(RecentCrashHandler.kt:53)
                      at org.pacien.tincapp.activities.common.RecentCrashHandler.access$sendReportMail(RecentCrashHandler.kt:31)
                      at org.pacien.tincapp.activities.common.RecentCrashHandler$handleRecentCrash$1.onClick(RecentCrashHandler.kt:41)
                      at android.support.v7.app.AlertController$ButtonHandler.handleMessage(AlertController.java:166)
                      at android.os.Handler.dispatchMessage(Handler.java:106)
                      at android.os.Looper.loop(Looper.java:193)
                      at android.app.ActivityThread.main(ActivityThread.java:6669)
                      at java.lang.reflect.Method.invoke(Native Method)
                      at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
                      at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
                   Caused by: android.os.TransactionTooLargeException: data parcel size 25364044 bytes
                      at android.os.BinderProxy.transactNative(Native Method)
                      at android.os.BinderProxy.transact(Binder.java:1127)
                      at android.app.IActivityManager$Stub$Proxy.startActivity(IActivityManager.java:3754)
                      at android.app.Instrumentation.execStartActivity(Instrumentation.java:1669)
                      at android.app.ContextImpl.startActivity(ContextImpl.java:917) 
                      at android.app.ContextImpl.startActivity(ContextImpl.java:888) 
                      at android.content.ContextWrapper.startActivity(ContextWrapper.java:379) 
                      at org.pacien.tincapp.context.App$Companion.sendMail(App.kt:82) 
                      at org.pacien.tincapp.context.App$Companion.sendMail$default(App.kt:74) 
                      at org.pacien.tincapp.activities.common.RecentCrashHandler.sendReportMail(RecentCrashHandler.kt:53) 
                      at org.pacien.tincapp.activities.common.RecentCrashHandler.access$sendReportMail(RecentCrashHandler.kt:31) 
                      at org.pacien.tincapp.activities.common.RecentCrashHandler$handleRecentCrash$1.onClick(RecentCrashHandler.kt:41) 
                      at android.support.v7.app.AlertController$ButtonHandler.handleMessage(AlertController.java:166) 
                      at android.os.Handler.dispatchMessage(Handler.java:106) 
                      at android.os.Looper.loop(Looper.java:193) 
                      at android.app.ActivityThread.main(ActivityThread.java:6669) 
                      at java.lang.reflect.Method.invoke(Native Method) 
                      at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493) 
                      at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858) 
E/org.pacien.tincapp.context.App: [main] Fatal application error.
                                  android.os.TransactionTooLargeException: data parcel size 25364044 bytes
                                  	at android.os.BinderProxy.transactNative(Native Method) ~[na:0.0]
                                  	at android.os.BinderProxy.transact(Binder.java:1127) ~[na:0.0]
                                  	at android.app.IActivityManager$Stub$Proxy.startActivity(IActivityManager.java:3754) ~[na:0.0]
                                  	at android.app.Instrumentation.execStartActivity(Instrumentation.java:1669) ~[na:0.0]
                                  	... 15 common frames omitted
                                  Wrapped by: java.lang.RuntimeException: Failure from system
                                  	at android.app.Instrumentation.execStartActivity(Instrumentation.java:1675) ~[na:0.0]
                                  	at android.app.ContextImpl.startActivity(ContextImpl.java:917) ~[na:0.0]
                                  	at android.app.ContextImpl.startActivity(ContextImpl.java:888) ~[na:0.0]
                                  	at android.content.ContextWrapper.startActivity(ContextWrapper.java:379) ~[na:0.0]
                                  	at org.pacien.tincapp.context.App$Companion.sendMail(App.kt:82) ~[na:0.0]
                                  	at org.pacien.tincapp.context.App$Companion.sendMail$default(App.kt:74) ~[na:0.0]
                                  	at org.pacien.tincapp.activities.common.RecentCrashHandler.sendReportMail(RecentCrashHandler.kt:53) ~[na:0.0]
                                  	at org.pacien.tincapp.activities.common.RecentCrashHandler.access$sendReportMail(RecentCrashHandler.kt:31) ~[na:0.0]
                                  	at org.pacien.tincapp.activities.common.RecentCrashHandler$handleRecentCrash$1.onClick(RecentCrashHandler.kt:41) ~[na:0.0]
                                  	at android.support.v7.app.AlertController$ButtonHandler.handleMessage(AlertController.java:166) ~[na:0.0]
                                  	at android.os.Handler.dispatchMessage(Handler.java:106) ~[na:0.0]
                                  	at android.os.Looper.loop(Looper.java:193) ~[na:0.0]
                                  	at android.app.ActivityThread.main(ActivityThread.java:6669) ~[na:0.0]
                                  	at java.lang.reflect.Method.invoke(Native Method) ~[na:0.0]
                                  	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493) ~[na:0.0]
                                  	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858) ~[na:0.0]
```

**Comment by pacien** [comment]
Due to `E/JavaBinder: !!! FAILED BINDER TRANSACTION !!!  (parcel size = 25364044)`.
Sending only the tail of the log should be enough.

**Comment by pacien** [comment]
Fixed in 4c1144b.
